pywbem/cim_obj.py

```diff
@@ -3653,33 +3701,43 @@ class CIMQualifierDeclaration(_CIMComparisonMixin):
         This variable will never be `None`.
 
       tosubclass (:class:`py:bool`):
-        If not `None`, specifies whether the qualifier value propagates
-        to subclasses.
+        If not `None`, specifies the tosubclass flavor. This flavor defines
+        whether the qualifier value propagates to subclasses. In MOF this
+        flavor is controlled by the `tosubclass` and `restricted` flavor
+        keywords.
```

Comment:
This attribute specifies not only the ToSubclass flavor, but in addition the Restricted flavor (by means of a value of `False`). I think we should describe that with explicit text for the values `True`, `False`, and `None`.